modules/core/03-connection/keeper/verify.go

```diff
@@ -159,9 +161,17 @@ func (k Keeper) VerifyPacketCommitment(
 		return sdkerrors.Wrapf(clienttypes.ErrClientNotActive, "client (%s) status is %s", clientID, status)
 	}
 
+	// calculate minimum block delay by dividing time delay period
+	// by the maximum expected time per block. Round up the block delay.
+	timeDelay := connection.GetDelayPeriod()
+	maxTimePerBlock := k.GetMaxTimePerBlock(ctx)
+	blockDelay := uint64(math.Ceil(float64(timeDelay) / float64(maxTimePerBlock)))
```

Comment:
I think this deserves a private helper function

Reply:
what happens if maxTimePerBlock is 0? I know that isn't a valid param, but in the test `params_test.go` you set it to 0 as a test. I guess the code would panic?

Reply:
That was a dumb test, removed. I'm assuming params are validated elsewhere